x-pack/solutions/observability/plugins/slo/server/lib/rules/slo_burn_rate/lib/build_query.ts

```diff
@@ -235,6 +246,7 @@ export function buildQuery(
         aggs: {
           ...buildWindowAggs(startedAt, slo, burnRateWindows, delayInSeconds),
           ...buildEvaluation(burnRateWindows),
+          ...buildTopHitsAgg(),
```

Comment:
nit: rename the function to communicate its intent not its internal details, e.g. buildGroupingAgg